libraries/botbuilder/src/cloudAdapter.ts

```diff
@@ -3,7 +3,7 @@
 
 import * as z from 'zod';
 import type { BotFrameworkHttpAdapter } from './botFrameworkHttpAdapter';
-import { Activity, CloudAdapterBase, InvokeResponse, StatusCodes, TurnContext } from 'botbuilder-core';
+import { Activity, Attachment, CloudAdapterBase, InvokeResponse, StatusCodes, TurnContext } from 'botbuilder-core';
```

Comment:
This import doesn't look to be necessary, going off of the changes in the file? Maybe it was accidentally left in? 😄